azure-pipeline-codeql.yml

```diff
@@ -22,7 +22,7 @@ stages:
       timeoutInMinutes: 90
       pool: 
         name: NetCore1ESPool-Internal
```

Comment:
Do the 1ES MMS images exist in both NetCore1ESPool-Internal and NetCore1ESPool-Internal-NoMSI?

Reply:
Yes